crates/trigger/src/cli.rs

```diff
@@ -58,19 +58,14 @@ where
     )]
     pub cache: Option<PathBuf>,
 
-    /// Print output for given component(s) to stdout/stderr
+    /// Print output to stdout/stderr only for given component(s)
     #[clap(
         name = FOLLOW_LOG_OPT,
         long = "follow",
```

Comment:
If an app has components A through D and someone specifies `--follow "A" "B" "C"`, does that automatically silence "D"?

Reply:
Yes.  `follow` is effectively "follow only these."  It comes from the previous opt-in model - it opted the nominated components in (whereas `follow-all` opted them all in, which is now replaced by the opt-out model of `--quiet`).  We could change it to be more opt-out, but the main use case for it is "I only want to see this one component that is giving me grief and not be distracted by anything else," so it's still the most concise way of doing that.  I hoped the reword would communicate that, but struggled with the placement of "only" and would be very happy to iterate if it's still unclear.

Reply:
I like the new pattern and think this is a great devx. I just wanted to clarify that it works as expected which is the case! Thanks for making this so clean